### ISSUE TITLE
Typed GraphQL support like NestJS

### ISSUE THREAD

**Issue by Archisman-Mridha** [SampaioLeal/destjs#19]
Add support for developing GraphQL apis using meta programming, like nestjs.

**Comment by SampaioLeal** [comment]
hi, can we talk more about this issue?
i'm not sure what is your intention with it and worried about the use case haha

this project started just for study, test and fun

i don't recommend the use in production...

**Comment by SampaioLeal** [comment]
but if you think that this project should continue, please share your thoughts :)

**Comment by Archisman-Mridha** [comment]
Yes you should continue this project. Later maybe you can approach deno later and merge with them. If you are successfull to achieve some percentage of what nestjs is like, then it will be a huge contribution to the deno community. I am excited if you can give graphql support like nestjs does with all the decorator stuff.

**Comment by SampaioLeal** [comment]
thanks for the words <3
but right now i guess i can't make this
i haven't used graphql with nestjs
i don't know how to make it, but we can study other implementations with deno and try something later

also, i can only work at weekends :(
